packages/gui/src/components/nfts/NFTDetails.tsx

```diff
@@ -231,6 +231,46 @@ export default function NFTDetails(props: NFTDetailsProps) {
       });
     }
 
+    if (metadata?.preview_image_uris) {
```

Comment:
Shouldn't the preview_image_uris/preview_video_uris support be in either NFTCard or NFTPreview?

Reply:
Disregard